src/lib/rest/rest.cpp

```diff
@@ -82,6 +83,20 @@ static unsigned int              threadPoolSize;
 
 /* ****************************************************************************
 *
+* correlatorGenerate - 
+*/
+static void correlatorGenerate(char* buffer)
```

Comment:
I've been searching for thread-safeness of the functions used from libuuid and nothing has come up.
From what the functions do, there is no reason to believer they aren't thread safe. Also, no complaints found on the web.
I think it's safe to assume we don't have a problem here. 

NTC